database/factories/ModelFactory.php

```diff
@@ -20,50 +19,33 @@
 |
 */
 
-// User Event Factory
-$factory->defineAs(Event::class, 'user', function (Generator $faker) {
-    return [
-        'northstar_id' => str_random(24),
-        'submission_type' => 'user',
-    ];
-});
-
 // Photo Factory
 $factory->define(Photo::class, function (Generator $faker) {
     return [
         'file_url' => 'https://s3.amazonaws.com/ds-rogue-test/uploads/reportback-items/12-1484929292.jpeg',
         'edited_file_url' => null,
         'caption' => $this->faker->sentence(),
-        'status' => 'pending',
-        'source' => 'phoenix-web',
-        'remote_addr' => '10.0.2.2',
     ];
 });
 
 // Post Factory
 $factory->define(Post::class, function (Generator $faker) {
     return [
-        'northstar_id' => str_random(24),
```

Comment:
Ya but it should be the same `northstar_id` that is on the signup the post relates to. If you generate it randomly in the factory, it would be different.